README.md

```diff
@@ -0,0 +1,21 @@
+# Development
+
+1. `echo DEBUG=True >> .env`
+2. `virtualenv env`
+3. `source env/bin/activate`
+4. `pip install -r requirements.txt`
+5. `npm install`
+6. In another terminal, install and run the database
+    1. Install postgres
+        * Mac: `brew install postgres`
+    2. Run postgres
+        * Mac: `brew services start postgresql`
```

Comment:
This is also found in the `brew info postgresql` documentation. If we wanted to suggest reading that?